src/util.ts

```diff
@@ -120,3 +120,22 @@ export function expandPath(input: string): string {
   const userHome = os.homedir()
   return input.replace(/\${userHome}/g, userHome)
 }
+
+/**
+ * Return the number of times a substring appears in a string.
+ * @param needle string
+ * @param haystack string
+ * @returns number
```

Comment:
Not too important, but the types seem redundant to me with TypeScript, idk why we had these on the above function, but I think generally we have not used these annotations

Reply:
Removed type annotations